<launch>
    <node pkg="video_handler" type="vhandler" name="vhandler" clear_params="true" output="screen">
        <param name="stream_from_file" value="true" />
        <param name="device_id" value="0" />
        <param name="filepath" value="/mnt/hgfs/Data/ants_videos/A8R10.M2TS" />
        <param name="topicname" value="/camera/image_raw" />
        <param name="framerate" value="30" />

        <param name="resize" value="true" />
        <param name="newWidth" value="960" />
        <param name="newHeigth" value="540" />
    </node>
</launch>
